routes/v1.js

```diff
@@ -76,4 +78,19 @@ apiRoutes.post(
   UserController.loginUser
 );
 
+apiRoutes.post(
+  '/resetpassword',
```

Comment:
@youngestdj this route can be better named, since it is not resetting the password but requesting for a password reset. This route name is better for the second route since it actually resets the password.
Ensure this change reflects in the controller method name too.

Reply:
I have renamed the route and controller names